crates/subspace-service/src/lib.rs

```diff
@@ -463,7 +463,11 @@ where
             backoff_authoring_blocks,
             subspace_link,
             can_author_with: CanAuthorWithNativeVersion::new(client.executor().clone()),
-            block_proposal_slot_portion: SlotProportion::new(2f32 / 3f32),
+            block_proposal_slot_portion: if cfg!(test) {
```

Comment:
When it is built as a dependency, `cfg!(test) == false`, so I think this should always be set to 98% unconditionally or it doesn't have any effect.

Reply:
You're right, it has no effect indeed, thanks for catching it! :see_no_evil: A feature gate in this crate is necessary....

Reply:
Indeed